packages/@glimmer/component/addon/-private/ember-component-manager.ts

```diff
@@ -11,7 +11,8 @@ import BaseComponentManager, {
 } from './base-component-manager';
 
 import GlimmerComponent, { Constructor } from './component';
-import { setDestroyed, setDestroying } from './destroyables';
+import * as destroyables from './destroyables';
+const { setDestroyed, setDestroying } = destroyables;
```

Comment:
I'm not sure I understand this change. shouldn't these two be equivalent? 🤔

Reply:
The original is a syntax error if `./destroyables` doesn't actually export either of those names. That happens in our implementation due to https://github.com/glimmerjs/glimmer.js/blob/8db4a62d1714bb47d15c09f3809cd8111cc07d43/packages/%40glimmer/component/ember-addon-main.js#L59-L63.

The new code avoids that problem by importing the whole namespace, and destructuring it *at runtime*. 

You can see the difference if you try an example directly in a browser.

Reply:
I see, thanks 👍